clubs/README.md

```diff
@@ -0,0 +1,202 @@
+[`workshops/`](../workshops/) directory) to get members started. More on how
+workshops work in [_The First Meeting_](#the-first-meeting).
+
+Throughout the school year, Hack Clubs go to (and sometimes host) hackathons,
+work on larger group projects, and---most importantly---build a thriving
+community of hackers at their high schools.
+
+In the below sections we'll walk you through the process of setting up your Hack
+Club and running your first few meetings. After you've gotten the first two
+meetings down, you should be set for the remainder of the school year :smile:.
+
+## Planning Your Hack Club
+
+You should plan on starting this process two to three weeks before having your
+first meeting. Fortunately there isn't an enormous amount of planning needed to
+start a club, but it's very important to get each of the following right.
+
+- [ ] Leadership team
+- [ ] Teacher sponsor and space
+- [ ] Club registration
+- [ ] 50 student sign-ups for the first meeting
+
+_(each of the below sections corresponds with a bullet point)_
+
+### Leadership Team
+
+You'll want 2-3 people on the primary leadership team running the club. When
+choosing your leadership team, we recommend working with close friends who you
+think have strong leadership skills. It's also a good idea to have a mix of
+genders on the leadership team. Correlation doesn't imply causation, but the
+club member demographics mirror the demographics of the leadership team in many
+clubs.
+
+Also, leadership skills are _far_ more important than technical skills. A common
+pitfall many clubs run into is selecting the leaders based on their technical
+skills rather than their leadership capacity.
+
+**Great clubs have been started by leaders that have never coded before.**
+
+When putting together the leadership team, make sure to choose one leader who's
+"in charge" (this will generally be you). This will be the person that has the
+final authority to make changes in the club.
+
+A common pitfall is to create a democratic selection process where everyone has
+an equal vote on decisions. This doesn't work because it's easy for less-active
+leaders to override decisions of the club president -- the best system is one
+where leaders get a say proportional to their investment in the club (and the
+easiest way to implement this is to have a fair leader).
+
+### Teacher Sponsor and Space
+
+A good teacher sponsor is your key to having access to regular space for club
+meetings and a good relationship with your school. Your teacher sponsor is the
+person who will regularly lend a classroom to you after-school for meetings. And
+when things go wrong with the school administration, your teacher sponsor will
+be your advocate. Choosing the wrong teacher sponsor has been the fatal to
+clubs, so be wary.
+
+The best teacher sponsors are personally invested in spreading coding to
+students. If your school started a computer science class (or another class, if
```

Comment:
I would change

> If your school started a computer science class (or another class, if it already has one), think about which teachers would volunteer to teach it.

To something like:

> This might be the teacher who would be the most likely to start a computer science class at your school (or the current computer science teacher at your school if you have one).